lib/iris/analysis/maths.py

```diff
@@ -332,7 +332,10 @@ def divide(cube, other, dim=None, in_place=False):
     _assert_is_cube(cube)
     other_unit = getattr(other, 'units', '1')
     new_unit = cube.units / other_unit
-    op = operator.idiv if in_place else operator.div
+    try:
+        op = operator.itruediv if in_place else operator.truediv
+    except AttributeError:
+        op = operator.idiv if in_place else operator.div
     return _binary_op_common(op, 'divison', cube, other, new_unit, dim,
```

Comment:
Just noticed there's a typo here; I don't know what that second parameter is used for though.